src/flashbar/collapsible-flashbar.tsx

```diff
@@ -0,0 +1,392 @@
+                  if (isFlashbarStackExpanded) {
+                    expandedItemRefs.current[getAnimationElementId(item)] = element;
+                  } else {
+                    collapsedItemRefs.current[getAnimationElementId(item)] = element;
+                  }
+                }}
+                style={
+                  !isFlashbarStackExpanded || transitioning
+                    ? {
+                        [customCssProps.flashbarStackIndex]:
+                          (item as StackableItem).collapsedIndex ?? (item as StackableItem).expandedIndex ?? index,
+                      }
+                    : undefined
+                }
+                key={getItemId(item)}
+              >
+                {showInnerContent(item) && (
+                  <Flash
+                    // eslint-disable-next-line react/forbid-component-props
+                    className={clsx(
+                      animateFlash && styles['flash-with-motion'],
+                      isVisualRefresh && styles['flash-refresh']
+                    )}
+                    key={getItemId(item)}
+                    ref={shouldUseStandardAnimation(item, index) ? transitionRootElement : undefined}
+                    transitionState={shouldUseStandardAnimation(item, index) ? state : undefined}
+                    {...item}
+                  />
+                )}
+              </li>
+            )}
+          </Transition>
+        ))}
+      </TransitionGroup>
+    </ul>
+  );
+
+  return (
+    <div
+      {...baseProps}
+      className={clsx(
+        baseProps.className,
+        styles.flashbar,
+        styles[`breakpoint-${breakpoint}`],
+        styles.stack,
+        isFlashbarStackExpanded && styles.expanded,
+        getVisualContextClassname('flashbar')
+      )}
+      ref={mergedRef}
+    >
+      <>
+        {isFlashbarStackExpanded && renderList()}
+        {isCollapsible && (
+          <div
+            className={clsx(
+              styles.toggle,
+              isVisualRefresh && styles['visual-refresh'],
+              isFlashbarStackExpanded ? styles.expanded : styles.collapsed,
+              transitioning && styles['animation-running']
+            )}
+            onClick={toggleCollapseExpand}
+            ref={toggleElementRef}
+          >
+            <span className={styles.status} role="status">
+              {toggleButtonText && <h2 className={styles.text}>{toggleButtonText}</h2>}
+              <span className={styles['types-count']} id={itemCountElementId}>
+                <LiveRegion>
+                  {counterTypes
+                    .map(
+                      ({ type, labelName }: { type: FlashType; labelName: LabelName }) =>
+                        `${countByType[type]} ${(i18nStrings && i18nStrings[labelName]) ?? ''}`
+                    )
+                    .join(', ')}
+                </LiveRegion>
+                <span aria-hidden="true" className={styles['types-count-visible']}>
```

Comment:
Why are we hiding these?

Reply:
This span is equivalent of what is in the LiveRegion.

Initially I put this whole span _inside_ the LiveRegion, with ARIA labels for the icons, but the LiveRegion would not read the labels, so went for doing two separate blocks, one for screen readers and one excluded from screen readers.

Reply:
Yeah I get it now. This might actually be a good case for a "native" ARIA live region. But this works, too, so no need to change it.